src/main/java/org/openmaptiles/Generate.java

```diff
@@ -604,16 +604,16 @@ private static List<OsmTableField> parseTableFields(Imposm3Table tableDefinition
    * "class") based on the "field mapping" defined in the layer schema definition.
    */
   static MultiExpression<String> generateFieldMapping(JsonNode valuesNode) {
-    MultiExpression<String> mapping = MultiExpression.of(new ArrayList<>());
+    List<MultiExpression.Entry<String>> mappings = new ArrayList<>();
```

Comment:
I want to change MultiExpression to make an immutable copy of the input, this code was assuming it is mutable.

Reply:
OK.